src/engine/SCons/Util.py

```diff
@@ -392,7 +392,7 @@ def MMM(m):
 try:
     BaseStringTypes = (str, unicode)
 except NameError:
-    BaseStringTypes = (str)
+    BaseStringTypes = str
```

Comment:
aren't we using as x in BaseStringTypes? so wouldn't this break that?

Reply:
if it must be a tuple, it would have to have been written as `(str,)` so as written they were useless parens. I can turn it into an actual tuple - see earlier comment on same.